File: test/java/NewOrderTest.java
import java.util.HashMap;

import Mexc.Sdk.*;

public class NewOrderTest {
    public static void main(String[] args) {
        String apiKey = "your apiKey";
        String apiSecret = "your apiSecret";
        Spot client = new Spot(apiKey, apiSecret);

        String symbol = "BTCUSDT";
        String side = "BUY";
        String orderType = "LIMIT";
        HashMap options = new HashMap<>();
        options.put("quantity", 5);
        options.put("price", 100);

        System.out.println("test new order" + client.newOrderTest(symbol, side, orderType, options));
    }
}

File: test/dotnet/NewOrderTest.cs
using System;
using System.Collections.Generic;
using Mexc.Sdk;
using Newtonsoft.Json;
namespace dotnet
{
   public class NewOrderTest
   {
       public static void Main(string[] args)
       {
           string apiKey = "your apiKey";
           string apiSecret = "your apiSecret";
           var spot = new Spot(apiKey, apiSecret);

           String symbol = "BTCUSDT";
           String side = "BUY";
           String orderType = "LIMIT";

           Dictionary<string, object> options = new Dictionary<string, object>();
           options.Add("quantity", 5);
           options.Add("price", 500);

           var newOrderTestRes = spot.NewOrderTest(symbol, side, orderType, options);
           Console.WriteLine("new order test");
           Console.WriteLine(JsonConvert.SerializeObject(newOrderTestRes));
       }

   }
}
